### PR TITLE
Hide the keyboard in a deferred task.

### PR DESCRIPTION
**Pull request by jason-simmons**
This is intended to eliminate the flicker that occurs when one widget hides
the keyboard and then another shows it within the same update cycle.

### CODE REVIEW

**Comment by Hixie** [comment]
LGTM

File: sky/packages/sky/lib/src/services/keyboard.dart [hunk 1]
```diff
@@ -43,6 +45,16 @@ class Keyboard {
     return _currentHandle;
   }
 
+  void _scheduleHide() {
```
Comment (abarth) [inline sky/packages/sky/lib/src/services/keyboard.dart]: Should we skip the scheduling if its already scheduled?

Comment (jason-simmons) [inline sky/packages/sky/lib/src/services/keyboard.dart]: Done

**Comment by abarth** [comment]
LGTM

**Pull request merged**